pretext/ExtraTopics/Pseudocode.ptx

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-pseudocode">
+  <title>Pseudocode</title>
+
+  <p><term>Pseudocode</term> is a mix of English and code (in our case, Python) that
+    programmers use to plan out their programs.</p>
+
+  <p>Writing pseudocode can help if you want to plan before you write
```

Comment:
Some more arguments I might make for pseudocode:
It is quicker to write than actual code.
It is readable to more people than actual code, leading to...
It can still be critiqued like regular code.

Reply:
Ok, I updated this one as well with in an unordered list form, so it may be a bit more awkward, but it has the info. Let me know what you think